### ISSUE TITLE
Cannot use 'broadFileSystemAccess'

### ISSUE THREAD

**Issue by kevin-jing** [MicrosoftDocs/windows-uwp#478]
Validation error. error C00CE169: App manifest validation error: The app manifest must be valid as per schema: Line 52, Column 21, Reason: 'broadFileSystemAccess' violates enumeration constraint of 'documentsLibrary picturesLibrary videosLibrary musicLibrary enterpriseAuthentication sharedUserCertificates userAccountInformation removableStorage appointments contacts phoneCall blockedChatMessages objects3D voipCall chat'. The attribute 'Name' with value 'broadFileSystemAccess' failed to parse.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 30a70676-883b-801e-2f07-881923cb0b96
* Version Independent ID: f968708e-33cd-ba83-fe9a-71672c961044
* Content: [App capability declarations - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations)
* Content Source: [windows-apps-src/packaging/app-capability-declarations.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/packaging/app-capability-declarations.md)
* Service: **unspecified**
* GitHub Login: @msatranjr
* Microsoft Alias: **misatran**

**Comment by kevin-jing** [comment]
My goal is to store a .db file to C:\Users\xxx\OneDrive\xxx. My UWP App read/write data with EFCore/SQLite. As OneDrive sync the file among one different device, so I always see the same data with my OneDrive account.

I used uap:Capability Name="documentsLibrary" and uap:Extension Category="windows.fileTypeAssociation" for both .json and .db files. The UWP App can access .json files, but one .db file.

So I want to explore broadFileSystemAccess, but encounter the error above.

**Comment by bryaan** [comment]
Would like to add using 

```
xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"

<rescap:Capability Name="broadFileSystemAccess" />
```

as showed in multiple msft docs and articles, simply does not work in VS2017.  

The namespace rescap is not recognized.

**Comment by kevin-jing** [comment]
Thanks, this issue is resolved. I'm use it normally. 👍

**Comment by Magic73** [comment]
So, how it can be solved?

I tryed bryaan solution but it doesn't works on VS2017.

**Comment by kevin-jing** [comment]
@Magic73 I just wrote a sample project: [https://github.com/kevin-jing/broadFileSystemAccess](url)

**Comment by amirvenus** [comment]
I wonder why this is not mentioned in the official docs:

https://docs.microsoft.com/en-us/windows/uwp/files/file-access-permissions

**Comment by saf-itpro** [comment]
@kevin-jing You may want to check the URL you provided above for your sample project. It does not work when clicking on the link directly or even if we right click and copy/paste the link to new window (it sends user to https://github.com/MicrosoftDocs/windows-uwp/issues/url).

But it works if we directly copy your link (by dragging the mouse on the link) and then paste it on a new tab of the browser. In other words, the text `https://github.com/kevin-jing/broadFileSystemAccess` is correct but the link behind it is incorrect.

**Comment by sialor** [comment]
It's still not work. I copied everything I need from the manifest file. The 'Capabilities' element contains an invalid 'Capability' child. 
![Снимок экрана 2021-02-09 162011](https://user-images.githubusercontent.com/39907387/107342490-d732e980-6af2-11eb-893d-f994893fe037.png)
UPD:
I solved my problem by enabling application access to the file system in OS settings. Settings -> Privacy -> Filesystem. [Xie Steven](https://stackoverflow.com/questions/55232209/package-appxmanifest-capabilites-addition-entry-is-failing) This solution did not get rid of 'invalid element' for me, but no UnauthorizedAccessException is thrown when trying to read and write files. An interesting note, the toggle switch in the settings OS is disabled if the line '<rescap: Capability Name =' broadFileSystemAccess '/>' is removed from '.appxmanifest'
